### PR TITLE
chore(deprecate): Remove unused create-cypress-tests package

### LINKED ISSUE #28457: Archive `create-cypress-tests` as it's not compatible with Cypress > 9.x
### Current behavior

Executing the npm package [create-cypress-tests](https://www.npmjs.com/package/create-cypress-tests) hangs at the step "Creating config files" after previously outputting a message referring to a missing file `node_modules/create-cypress-tests/dist/initial-template/plugins/index.js`.

It appears that [npm/create-cypress-tests](https://github.com/cypress-io/cypress/tree/develop/npm/create-cypress-tests) was never converted to run on non-legacy Cypress versions > `9.x` including component testing in its GA version which was released with Cypress [11.0.0](https://docs.cypress.io/guides/references/changelog#11-0-0).

The Migration guide includes the advice [Plugins File Removed](https://docs.cypress.io/guides/references/migration-guide#Plugins-File-Removed).

### Desired behavior

[create-cypress-tests](https://www.npmjs.com/package/create-cypress-tests) should set up a working Cypress test environment for Cypress `13.x` e2e and component testing.

OR

The package [create-cypress-tests](https://www.npmjs.com/package/create-cypress-tests) should be retired.

### Test code to reproduce

```shell
mkdir cy-create-cypress-tests
cd cy-create-cypress-tests
npm install yarn@latest -g
yarn init -y
npm install create-cypress-tests@latest -g
npx create-cypress-tests
```

### Cypress Version

Cypress `13.6.0`
`create-cypress-tests@2.0.4`

### Node version

`v20.10.0`

### Operating System

Ubuntu `22.04.3 LTS`

### Debug Logs

```shell
✔ Installing cypress (yarn add cypress --dev)
⠋ Creating config files[Error: ENOENT: no such file or directory, stat '/home/mike/n/lib/node_modules/create-cypress-tests/dist/initial-template/plugins/index.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'stat',
  path: '/home/mike/n/lib/node_modules/create-cypress-tests/dist/initial-template/plugins/index.js'
}
⠦ Creating config files
```

### Other

_No response_

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- Closes https://github.com/cypress-io/cypress/issues/28457

### Additional details

This package is no longer used or referenced in our codebase since v10 (see https://github.com/cypress-io/cypress/pull/21438/files). We are running tests within this package, which is using some resources. Cleaning up this unused package. 

Screen Shot 2023-12-06 at 12 49 46 PM

### Steps to test

Hopefully I didn't mis remove something in the build files. Tests should pass.

### How has the user experience changed?

We will deprecate the create-cypress-tests npm package which has like ~7 dls per week, so don't see any impact.

### PR Tasks

- N/A Have tests been added/updated?
- N/A Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- N/A Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -92,10 +92,6 @@ system-tests/lib/fixtureDirs.ts
 /packages/frontend-shared/src/generated
 /packages/frontend-shared/cypress/e2e/support/e2eProjectDirs.ts
 
-# from npm/create-cypress-tests
-/npm/create-cypress-tests/initial-template
-/npm/create-cypress-tests/src/test-output
```
Comment (emilyrohrbough) [inline .gitignore]: might be worth keeping this around so stale files don't accidentally get checked in when people change branches?

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Need to remove the required status check.
Screen Shot 2023-12-06 at 12 07 38 PM

File: .gitignore (hunk 1 above)
Comment (jennifer-shehane) [reply .gitignore]: https://github.com/cypress-io/cypress/pull/28472/commits/2b158f5b838f946198cbd7ec3724b27e198dd8bd

**Comment by jennifer-shehane** [comment]
I removed the required status checks for npm-create-cypress-tests

**Review by rachelruderman — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@jennifer-shehane 

> How has the user experience changed?
> We will deprecate the create-cypress-tests npm package which has like ~7 dls per week, so don't see any impact.

According to the npm documentation for [unpublish](https://docs.npmjs.com/unpublishing-packages-from-the-registry), you could `unpublish` instead of deprecating, since it meets the criteria:

[Packages published more than 72 hours ago](https://docs.npmjs.com/policies/unpublish#packages-published-more-than-72-hours-ago)
>Regardless of how long ago a package was published, you can unpublish a package that meets all of the following conditions:

>
> - no other packages in the npm Public Registry depend on it
> - it had less than 300 downloads over the last week
> - it has a single owner/maintainer

**Comment by MikeMcC399** [comment]
- This PR seems to have caused issue #28504

**Pull request merged**